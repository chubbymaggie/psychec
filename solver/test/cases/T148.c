void f() {
    T1 a;
    T2 b;
    b.y = 10;
    a.x = b;
}

void g() {
    T3 c;
    T4 d;
    d.w = c;
    c.z = 10;
}

void h() {
    T5 m;
    T6 n;
    n.yy = 10;
    m->xx = n;
}

void i() {
    T7 p;
    T8 q;
    q->ww = p;
    p.zz = 10;
}


void k() {
    T9 ab;
    ab->cd = 10;
    T10* pab = ab;
    T11 ab2 = ab;
}

void j() {
    T12 rt;
    rt->qs = 10;
    T13 rt2 = rt;
}

void l() {
    struct Elab elab;
    elab.val = 10;
    ElabAlias alias = elab;

    struct Bale* bale;
    bale->lav = 3.14;
}
