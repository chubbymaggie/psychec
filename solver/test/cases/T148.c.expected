/* Forward declarations */
typedef  struct TYPE_23__   TYPE_9__ ;
typedef  struct TYPE_22__   TYPE_8__ ;
typedef  struct TYPE_21__   TYPE_7__ ;
typedef  struct TYPE_20__   TYPE_6__ ;
typedef  struct TYPE_19__   TYPE_5__ ;
typedef  struct TYPE_18__   TYPE_4__ ;
typedef  struct TYPE_17__   TYPE_3__ ;
typedef  struct TYPE_16__   TYPE_2__ ;
typedef  struct TYPE_15__   TYPE_1__ ;
typedef  struct TYPE_14__   TYPE_13__ ;

/* Type definitions */
struct Elab {int val; } ;
struct Bale {double lav; } ;
struct TYPE_23__ {int qs; } ;
struct TYPE_22__ {int y; } ;
struct TYPE_21__ {int z; } ;
struct TYPE_20__ {TYPE_7__ w; } ;
struct TYPE_18__ {int yy; } ;
struct TYPE_19__ {TYPE_4__ xx; } ;
struct TYPE_17__ {int zz; } ;
struct TYPE_16__ {TYPE_3__ ww; } ;
struct TYPE_15__ {int cd; } ;
struct TYPE_14__ {TYPE_8__ x; } ;
typedef  TYPE_1__* T9 ;
typedef  TYPE_2__* T8 ;
typedef  TYPE_3__ T7 ;
typedef  TYPE_4__ T6 ;
typedef  TYPE_5__* T5 ;
typedef  TYPE_6__ T4 ;
typedef  TYPE_7__ T3 ;
typedef  TYPE_8__ T2 ;
typedef  TYPE_9__* T13 ;
typedef  TYPE_9__* T12 ;
typedef  TYPE_1__* T11 ;
typedef  TYPE_1__ T10 ;
typedef  TYPE_13__ T1 ;
typedef  struct Elab ElabAlias ;

/* Variables and functions */

